### PR TITLE
Features and fixes

### PR DESCRIPTION
**Pull request by frankrousseau**
Here are many changes to the new home. I will open issues for it soon for the PO review.

* simplify device line
* improve display of buttons inside notification sidebar
* fix some style and responsive
* fix current app title when the user acccess to the app panel through update notifs
* make internal apps look like apps (no background)
* fix styles of dismiss all button
* allow to display app logs (add a button show logs to the app panel)
* add route to send an email from the user to support
* add a widget to send an email to support
* make back button more explicit

### CODE REVIEW

File: client/app/styles/main.styl [hunk 1]
```diff
@@ -392,11 +388,11 @@ body
             background-color orange + 10%
 
 
-@media screen and (-webkit-min-device-pixel-ratio:0)
+@media screen and (-webkit-min-device-pixel-ratio: 0)
```
Comment (m4dz) [inline client/app/styles/main.styl]: Why do we use a `-webkit` property here? Do we target only specific browsers for a hack with this set of rules?

**Comment by m4dz** [comment]
Just for information, I don't know why we use an image for the back button instead of a FontAwesome icon (which keep us from maintaining this icon ourselves and prevent an extra request)?

File: client/app/styles/main.styl (hunk 1 above)
Comment (frankrousseau) [inline client/app/styles/main.styl]: I think it was there by default.

**Comment by frankrousseau** [comment]
Images are prettier than font awesome in most cases. I will look if I find something similar in FA later.

**Comment by frankrousseau** [comment]
I need to merge this one. There is too much changes waiting and the PO still can't review what I did.

**Comment by m4dz** [comment]
all right :)

**Pull request merged**